geowebcache/web/src/main/webapp/WEB-INF/geowebcache-wmtsservice-context.xml

```diff
@@ -22,5 +22,6 @@
     <constructor-arg ref="gwcURLMangler"/>
     <constructor-arg ref="geowebcacheDispatcher"/>
     <property name="securityDispatcher" ref="gwcSecurityDispatcher"/>
+  <property name="mainConfiguration" ref="gwcXmlConfig"/>
```

Comment:
Indent

Reply:
Ouch ... the file is using a mixture of tabs \ spaces, I'm reformatting everything to spaces,